src/widget/genericchatroomwidget.cpp

```diff
@@ -72,6 +72,7 @@ void GenericChatroomWidget::compactChange(bool _compact)
     // avatar
     if (isCompact())
     {
+        delete textLayout;  // Not needed
```

Comment:
You can move recreation of textLayout (c'tor in line 63) here and replace the delete - this is safe. As I'm not sure what that nasty comment on the above delete (line 59) means I'd leave it untouched for now.